### PR TITLE
Iamxy/dockerfile

### PR DESCRIPTION
**Pull request by iamxy**
@siddontang @BusyJay @andelf @tennix PTAL

### CODE REVIEW

File: docker/rust_dockerfile [hunk 1]
```diff
@@ -1,28 +1,60 @@
...
+	&& rm golang.tar.gz
+
+ENV GOPATH /go
+ENV GOROOT /usr/local/go
+ENV PATH $GOPATH/bin:$GOROOT/bin:$PATH
+
+RUN go get golang.org/x/tools/go/gcimporter15 \
+	&& go get github.com/golang/lint/golint
+
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --no-modify-path --default-toolchain nightly-2017-03-27 -y \
```
Comment (andelf) [inline docker/rust_dockerfile]: use "2017-05-29"

File: docker/rust_dockerfile [hunk 2]
```diff
@@ -1,28 +1,60 @@
...
+	&& make install \
+	&& rm -rf /gflags.tar.gz /gflags-2.1.2
+
+RUN curl -L https://github.com/facebook/rocksdb/archive/v5.4.5.tar.gz -o rocksdb.tar.gz \
+	&& tar xf rocksdb.tar.gz \
+	&& cd rocksdb-5.4.5 \
+	&& make -j install-shared \
+	&& rm -rf /rocksdb.tar.gz /rocksdb-5.4.5
+
+ENV GOLANG_VERSION 1.8
```
Comment (iamxy) [inline docker/rust_dockerfile]: Indeed...

Comment (iamxy) [inline docker/rust_dockerfile]: `make static_release` need Go environment.

**Comment by siddontang** [comment]
LGTM

**Comment by andelf** [comment]
LGTM

**Pull request merged**